Tests/FunctionalTests/Sources/EarlGrey.swift

```diff
@@ -39,7 +39,7 @@ public func GREYAssertTrue(@autoclosure expression: () -> BooleanType, reason: S
 public func GREYAssertFalse(@autoclosure expression: () -> BooleanType, reason: String) {
   GREYAssert(!expression().boolValue,
     reason,
-    details: "Expected the boolean expression to be true")
```

Comment:
There are 5 copies floating around. Can we symlink them?

1. Demo/EarlGreyExample/EarlGreyExampleSwiftTests/EarlGrey.swift
2. Tests/FunctionalTests/Sources/EarlGrey.swift
3. docs/downloads/EarlGrey.swift
4. gem/lib/earlgrey/files/EarlGrey.swift
5. gem/spec/fixtures/carthage_after/AutoEarlGrey/EarlGrey.swift